### PR TITLE
feat: Units input element consistency

### PR DESCRIPTION
**Pull request by eliotwrobson**
Add attributes and change styling of units input element for consistency with other elements. Screenshots below:

New feedback formatting (less weird than before):
image

Invalid on question panel:
image

Score on question panel:
image

New invalid button on submission panel:
image

EDIT:
In detail, I've added the `show-score` attribute, which will hide the score if set to false. The other changes are cosmetic as detailed in the above screenshots.

### CODE REVIEW

File: apps/prairielearn/elements/pl-units-input/pl-units-input.mustache [hunk 1]
```diff
@@ -90,14 +95,19 @@
...
-    <span class="badge text-danger badge-invalid">
-        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Invalid
-    </span>
-    <a href="javascript:void(0);" role="button" class="btn btn-sm btn-secondary small border"
-        data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true"
-        style="margin-left: 5px" title="Format Error" tabindex="0" data-content="{{parse_error}}">
-        Why
-        <i class="fa fa-question-circle" aria-hidden="true"></i>
+    <a
+        href="javascript:void(0);"
```
Comment (nwalters512) [inline apps/prairielearn/elements/pl-units-input/pl-units-input.mustache:99]: Why this? Is this just copy-pasta?

**Review by nwalters512 — APPROVED** [review]
Seems reasonable!

File: apps/prairielearn/elements/pl-units-input/pl-units-input.mustache (hunk 1 above)
Comment (eliotwrobson) [reply apps/prairielearn/elements/pl-units-input/pl-units-input.mustache:99]: Yep, copy pasta

File: apps/prairielearn/elements/pl-units-input/pl-units-input.mustache [hunk 2]
```diff
@@ -90,14 +95,19 @@
...
+        role="button"
+        class="badge text-danger badge-invalid btn btn-sm btn-secondary small border"
+        data-placement="auto"
+        data-trigger="focus"
+        data-toggle="popover"
+        data-html="true"
+        style="margin-left: 5px"
```
[the remark this answers is not in the document]
Comment (eliotwrobson) [reply apps/prairielearn/elements/pl-units-input/pl-units-input.mustache:106]: Also copy-pasta, will change!

[the remark this answers is not in the document]
Comment (eliotwrobson) [reply apps/prairielearn/elements/pl-units-input/pl-units-input.mustache]: Removing this didn't seem to have any effect on how the button was displayed, so I didn't bother to add back the bootstrap class https://github.com/PrairieLearn/PrairieLearn/pull/8189/commits/484ac577be783015962a75abfe095c2a6e1b8f43

**Pull request merged**